files/css/site.css

```diff
@@ -2,6 +2,7 @@ h1 { font-size: 2.2rem; font-weight: bold; }
 h2 { font-size: 1.8rem; font-weight: bold; }
 h3 { font-size: 1.6rem; font-weight: 600; }
 h4,h5,h6 { color: #222222; font-size: 1.4rem; font-weight: 600; }
+h1,h2,h3,h4,h5,h6 { margin-top: 2rem; }
```

Comment:
Nit: Not sure what this is changing w/o actually looking at the result, but if it is what I think it is (vertical margins should match line height) then you want the `margin-top` folded into the existing `h1 ... h6` above and use a `margin-top` equal to the appropriate `font-size`.